src/simtools/schemas/model_parameters/flasher_var_photons.schema.yml

```diff
@@ -1,15 +1,15 @@
 %YAML 1.2
 ---
-title: Schema for laser_var_photons model parameter
+title: Schema for flasher_var_photons model parameter
 schema_version: 0.1.0
 meta_schema: simpipe-schema
 meta_schema_url: https://raw.githubusercontent.com/gammasim/simtools/main/src/simtools/schemas/model_parameter_and_data_schema.metaschema.yml
 meta_schema_version: 0.1.0
-name: laser_var_photons
+name: flasher_var_photons
 description: |-
-  Relative variation of laser shots from shot to shot, independent for each telescope.
+  Relative variation of flasher shots from shot to shot.
```

Comment:
I assume it's the variance of the number of photons? Perhaps the description should say this